testdata/phase-2-plugins/simple-external-plugin/main.go

```diff
@@ -0,0 +1,328 @@
+/*
```

Comment:
@everettraven that is great. 
However, we cannot add under testdata
Why?
All under this directory is removed and generated programmatically, see:
https://github.com/kubernetes-sigs/kubebuilder/blob/master/Makefile#L69-L72

**So, where to add the sample?**
What about we have dir for we use the sample on the docs as tutorial for how to write your own plugin?
That would be like: https://github.com/kubernetes-sigs/kubebuilder/tree/master/docs/book/src/component-config-tutorial

And we could add here:
https://github.com/kubernetes-sigs/kubebuilder/tree/master/docs/book/src/plugins

Therefore we can use this code in a tutorial on the docs/ WDYT?

Reply:
@camilamacedo86 that makes sense! I can work on moving it!